templates/experiments/components/pagination_buttons.html

```diff
@@ -0,0 +1,32 @@
+{% if total_pages > 1 %}
+  <div class="flex {% if btn_size == 'sm' %}gap-1{% else %}justify-right{% endif %}">
+    {% if page > 1 %}
+      <a href="#table-top"
+         class="btn {% if btn_size == 'sm' %}btn-sm{% endif %}"
+         hx-get="{% url 'experiments:experiment_session_messages_view' request.team.slug experiment.public_id experiment_session.external_id %}?page=1{% if search %}&search={{ search }}{% endif %}"
```

Comment:
Would it work to use the [querystring](https://docs.djangoproject.com/en/5.1/ref/templates/builtins/#querystring) tag for these?

```suggestion
         hx-get="{% url 'experiments:experiment_session_messages_view' request.team.slug experiment.public_id experiment_session.external_id %}{% querystring page=1 search=search %}"
```